### PR TITLE
update installation guides for sylius 1.8

### PR DESCRIPTION
**Pull request by arti0090**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,60 @@ It is used by `KnpSnappyBundle` and can be configured according to [their documm
     mkdir -p templates/bundles/SyliusAdminBundle/
     cp -R vendor/sylius/refund-plugin/src/Resources/views/SyliusAdminBundle/* templates/bundles/SyliusAdminBundle/
     ```
+   
+4. If you use sylius v1.8 you also need to change files `src/Entity/Shipping/Shipment.php` and `src/Entity/Order/Adjustment.php` to use proper traits and interfaces
```
Comment (GSadee) [inline README.md:60]:
```suggestion
4. If you use Sylius v1.8 you also need to change files `src/Entity/Shipping/Shipment.php` and `src/Entity/Order/Adjustment.php` to use proper traits and interfaces:
```

File: README.md [hunk 2]
```diff
@@ -56,6 +56,60 @@ It is used by `KnpSnappyBundle` and can be configured according to [their documm
...
+   * @ORM\Table(name="sylius_shipment")
+   */
+   class Shipment extends BaseShipment implements RefundShipmentInterface
+   {
+   use ShipmentTrait;
+
+      public function __construct()
+      {
+         parent::__construct();
+         /** @var ArrayCollection<array-key, BaseAdjustmentInterface> $this->adjustments */
```
Comment (GSadee) [inline README.md:108]: `BaseAdjustmentInterface` is not declared anywhere if I see correctly

File: docs/legacy_installation.md [hunk 3]
```diff
@@ -51,6 +51,60 @@
     cp -R vendor/sylius/refund-plugin/src/Resources/views/SyliusAdminBundle/* templates/bundles/SyliusAdminBundle/
     ```
 
+1. If you use sylius v1.8 you also need to change files `src/Entity/Shipping/Shipment.php` and `src/Entity/Order/Adjustment.php` to use proper traits and interfaces
```
Comment (GSadee) [inline docs/legacy_installation.md:54]:
```suggestion
1. If you use Sylius v1.8 you also need to change files `src/Entity/Shipping/Shipment.php` and `src/Entity/Order/Adjustment.php` to use proper traits and interfaces:
```

File: docs/legacy_installation.md [hunk 4]
```diff
@@ -51,6 +51,60 @@
...
+   * @ORM\Table(name="sylius_shipment")
+   */
+   class Shipment extends BaseShipment implements RefundShipmentInterface
+   {
+   use ShipmentTrait;
+
+      public function __construct()
+      {
+         parent::__construct();
+         /** @var ArrayCollection<array-key, BaseAdjustmentInterface> $this->adjustments */
```
Comment (GSadee) [inline docs/legacy_installation.md:102]: As above

File: README.md (hunk 2 above)
Comment (arti0090) [reply README.md:108]: Well but it works, also same thing happens in Plus 🧙

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, @arti0090! :1st_place_medal:

**Pull request merged**